deps.bzl

```diff
@@ -384,3 +384,19 @@ genrule(
 )
             """,
         )
+
+    if "canton" not in native.existing_rules():
+        http_archive(
+            name = "canton",
+            build_file_content = """
+        package(default_visibility = ["//visibility:public"])
+
+        java_import(
+            name = "lib",
+            jars = glob(["lib/**/*.jar"]),
+        )
+        """,
+            sha256 = "9fb38c922d302c94e200c161ec8467ea39f4c31c9a7fad0b32f1493170a51fa7",
+            strip_prefix = "canton-community-0.23.0-SNAPSHOT",
+            urls = ["https://www.canton.io/releases/canton-community-20210407.tar.gz"],
```

Comment:
Is there a good reason why there is a mismatch between the version in the url and the prefix in the tarball? that seems a bit confusing.

Reply:
This is an artifact of canton doing nightly releases instead of as-needed snapshots.

Reply:
What’s stopping you from using the date in the prefix as well?

Reply:
Makes sense.